kfs-web/start-tomcat.sh

```diff
@@ -2,4 +2,4 @@
 
 export EXTERNAL_CONFIG="-Dadditional.kfs.config.locations=$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-config.properties,$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-rice-config.properties,$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-security-config.properties,$UA_KFS_INSTITUTIONAL_CONFIG_DIR/institutional-config.properties"
 
-mvn $EXTERNAL_CONFIG -DskipTests=true -P oracle -Dserver.name="http://localhost:8080" tomcat7:run-war
+mvn -DLOG4J_XML_PATH="$HOME/logs" $EXTERNAL_CONFIG -DskipTests=true -P oracle -Dserver.name="http://localhost:8080" tomcat7:run-war
```

Comment:
This change was made by Will. I asked him about it and he said, "`-DLOG4J_XML_PATH` needs to have a value, and before it didn't (this feeds into the logging improvements made last fall)" It's for the "old" way to run KFS locally, documented in Confluence here: https://confluence.arizona.edu/display/KFS5Up/How+to+Setup+Developer+Workstation#HowtoSetupDeveloperWorkstation-PSAAlternative(BeforeYouGoFurther)